plugins/@grouparoo/demo/config/shared/all/identity/apps/my_calculated-property_app.json

```diff
@@ -0,0 +1,7 @@
+{
+  "class": "App",
+  "id": "demo_calculated-property_app",
```

Comment:
It feels a little strange that this is a mix of `_` and `-` but I'm assuming we do this elsewhere.

Reply:
Good point!  Let's do all snake casing, I'll update the docs to match.  This is our first multi-word plugin!